main/libpaper/template.py

```diff
@@ -1,32 +1,20 @@
 pkgname = "libpaper"
-pkgver = "1.1.29"
+pkgver = "2.2.5"
 pkgrel = 0
 build_style = "gnu_configure"
 make_cmd = "gmake"
 hostmakedepends = ["pkgconf", "gmake", "automake", "libtool", "gettext"]
 pkgdesc = "Library for handling paper characteristics"
 maintainer = "q66 <q66@chimera-linux.org>"
-license = "GPL-2.0-or-later"
-url = "http://packages.debian.org/unstable/source/libpaper"
-source = f"$(DEBIAN_SITE)/main/libp/{pkgname}/{pkgname}_{pkgver}.tar.gz"
-sha256 = "26330e21e9a3124658d515fd850b0cde546ff42d89b2596a5264c5f1677f0547"
+license = "MIT AND GPL-2.0-only AND GPL-3.0-or-later AND custom:none"
+url = "https://github.com/rrthomas/libpaper"
+source = f"{url}/releases/download/v{pkgver}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "7be50974ce0df0c74e7587f10b04272cd53fd675cb6a1273ae1cc5c9cc9cab09"
 
 
 def post_install(self):
+    self.install_license("COPYING-MIT")
     self.install_dir("etc/libpaper.d", empty=True)
-    # systemwide default papersize
-    with (self.destdir / "etc/papersize").open("w") as ps:
-        ps.write("# Write the paper size here, see papersize(5)")
-    # localization
-    for f in (self.cwd / "debian/po").glob("*.po"):
-        loc = f"usr/share/locale/{f.stem}/LC_MESSAGES"
-        self.install_dir(loc)
-        self.do(
-            "msgfmt",
-            "-o",
-            self.chroot_destdir / loc / "libpaper.mo",
-            self.chroot_cwd / "debian/po" / f.name,
-        )
```

Comment:
these files were moved to the debian package repo (https://github.com/rrthomas/libpaper/commit/671171dfe2e5469bbd5b0433ac7cac04e5f8a542#diff-85bfc337e1efbd04947483a4c68ee70d49f07c9701862f371932730b0e62f9df), not sure whether to poke upstream about it